packages/core/src/routes/interaction/utils/social-verification.ts

```diff
@@ -41,11 +40,10 @@ export const createSocialAuthorizationUrl = async (
 
 export const verifySocialIdentity = async (
   { connectorId, connectorData }: SocialConnectorPayload,
-  ctx: Context,
+  ctx: WithLogContext,
   provider: Provider
 ): Promise<SocialUserInfo> => {
-  // eslint-disable-next-line prefer-destructuring, @typescript-eslint/no-unsafe-assignment
-  const createLog: LogContext['createLog'] = ctx.createLog;
+  const { createLog } = ctx;
   const log = createLog('Interaction.SignIn.Identifier.Social.Submit');
```

Comment:
just `ctx.createLog()`?

Reply:
updated with all the rest places using a same syntax